src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -182,6 +182,34 @@ public override async Task<IEnumerable<StorageListItem>> ListAsync(CancellationT
             return files.Select(GetStorageListItem).AsEnumerable();
         }
 
+        public override async Task<bool> UpdateCacheControlAsync(Uri resourceUri, string cacheControl, CancellationToken cancellationToken)
+        {
+            string blobName = GetName(resourceUri);
+            CloudBlockBlob blob = GetBlockBlobReference(blobName);
+
+            await blob.FetchAttributesAsync(
+                accessCondition: null,
+                options: null,
+                operationContext: null,
+                cancellationToken: cancellationToken);
+
+            if (blob.Properties.CacheControl != cacheControl)
+            {
+                blob.Properties.CacheControl = cacheControl;
+
+                await blob.SetPropertiesAsync(
+                    accessCondition: AccessCondition.GenerateIfMatchCondition(blob.Properties.ETag),
+                    options: null,
+                    operationContext: null,
+                    cancellationToken);
```

Comment:
Why is this line different from line 194? 

```suggestion
                    cancellationToken: cancellationToken);
```

Reply:
I suggest we remove repetition from line 194 instead (if compiler is happy, of course).

Reply:
I removed the named parameter in these cases since it's clear.